models/api.models.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db/connection');
+const fs = require("fs/promises");
+
+const readApi = () => {
+    return fs.readFile('./endpoints.json', 'utf-8')
+    .then((data) => {
+        return data;
+    });
+};
```

Comment:
Nice this logic is a good way to go about this but It should be in the controller rather than the model as we are not interacting with the database.